### PR TITLE
Google Drive: send email when material is finished uploading

### PR DESCRIPTION
**Pull request by bseeger**
https://www.pivotaltracker.com/story/show/187120829

### CODE REVIEW

File: src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java [hunk 1]
```diff
@@ -34,24 +38,42 @@ public class TransmissionCommands {
     @Value("${google.drive.queen-annes-county-directory-id}")
     private String QUEENANNES_COUNTY_GOOGLE_DIR_ID;
 
+    private List<String> emailRecipients_Baltimore =
+            List.of(
+                    "bseeger+balt@codeforamerica.org",
```
Comment (vrajmohan) [inline src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java:43]: These may or may not be considered PII, we should verify.

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -928,3 +928,7 @@ email-to-recipient.subject=Application Received
 email-to-recipient.body=<html><body><p>We received your Maryland Benefits application. Because you applied for food and/or cash programs, \
   you are automatically considered for an energy benefit (OHEP).</p><p>Application status: In review</p><p>Your county will contact you \
   via mailing address and/or phone number with next steps.</p><p>If you want an update on your case, please call your county.</p></body></html>
+
+email-to-caseworkers.subject=new application case #{0}
+email-to-caseworkers.body=<html><body><p>A new benefit application has been submitted. Please go to the following link to retrieve the application of case number #{0} <a target="_blank" href="{1}">{1}</a></p> \
```
Comment (vrajmohan) [inline src/main/resources/messages.properties:933]: It should be confirmation number - case number has a distinct meaning.

File: src/main/java/org/mdbenefits/app/config/TransmissionConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "transmission")
+public class TransmissionConfiguration {
+
+    Map<String, String> googleDriveDirectoryId;
```
Comment (bseeger) [inline src/main/java/org/mdbenefits/app/config/TransmissionConfiguration.java:11]: @spokenbird I'm not really sure what value this config class adds. I know I'm not tracking the variables correctly anymore and things still seem to work fine w/o this functioning.  (I think).  Why do I need this class?

File: src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java [hunk 4]
```diff
@@ -171,12 +190,40 @@ private void transmitBatch(List<Transmission> transmissions) {
...
 
-            // TODO - send emails to state (another ticket)
+            sendEmailToCaseworkers(transmission, confirmationNumber, emailRecipients, newFolder.getUrl(), errorMap);
 
             updateTransmissionStatus(transmission, TransmissionStatus.COMPLETED, errorMap, true);
         });
     }
 
+    // email addresses is a string of email addresses with a space in between
+    private void sendEmailToCaseworkers(Transmission transmission, String confirmationNumber, String emailAddresses,
```
Comment (spokenbird) [inline src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java:200]: This has email address as the third parameter but above it's passed as newFolder.gerUrl()?

File: src/main/java/org/mdbenefits/app/config/TransmissionConfiguration.java (hunk 3 above)
Comment (spokenbird) [reply src/main/java/org/mdbenefits/app/config/TransmissionConfiguration.java:11]: My understanding is it sets the prefix as valid in application.yaml and enables autocompletion. Without it, the application.yaml won't register the values you add there. Just tried removing it and got:

`Could not resolve placeholder 'BALTIMORE_COUNTY_EMAIL_ADDRESSES' in value "${BALTIMORE_COUNTY_EMAIL_ADDRESSES}"`

Comment (spokenbird) [reply src/main/java/org/mdbenefits/app/config/TransmissionConfiguration.java:11]: It was allowing us to have multi keys before a field like foo.bar.thingWeAdd=value, but in this case if it's just `transmission` then I don't think we need it.

**Comment by spokenbird** [comment]
Looks good to me pending the one comment.

File: src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java (hunk 4 above)
Comment (bseeger) [reply src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java:200]: Line 193 looks correct to me.  It's sending `emailRecipients` as the third parameter.

**Pull request merged**